pkg/services/sqlstore/annotation_cleanup_test.go

```diff
@@ -1,3 +1,5 @@
+// +build integration
```

Comment:
We usually dont add this build flag even if the tests writes/reads from the database. The unit tests use an inmemory sqlite which is fast enough for tests.

Reply:
OK, I can remove it. It was useful for testing against different backends locally.